tests/client/test_client_task_meilisearch.py

```diff
@@ -91,3 +92,30 @@ def test_get_task_inexistent(client):
     """Tests getting a task that does not exists."""
     with pytest.raises(Exception):
         client.get_task("abc")
+
+
+def test_cancel_tasks(client):
+    """Tests cancel a task with uid 1."""
+    task = client.cancel_tasks({"uids": ["1"]})
+    tasks = client.get_tasks({"types": "taskCancelation"})
+
+    assert isinstance(task, TaskInfo)
+    assert task.task_uid is not None
+    assert task.index_uid is None
+    assert task.status == "enqueued" or "processing" or "succeeded"
+    assert task.type == "taskCancelation"
+    assert "uids" in tasks["results"][0]["details"]["originalFilters"]
+
+def test_cancel_every_task(client):
+    """Tests cancel every task."""
+    task = client.cancel_tasks({"statuses": ["enqueued", "processing"]})
+    tasks = client.get_tasks({"types": "taskCancelation"})
+
+    assert isinstance(task, TaskInfo)
+    assert task.task_uid is not None
+    assert task.index_uid is None
+    assert task.status == "enqueued" or "processing" or "succeeded"
```

Comment:
```suggestion
    assert task.status == "enqueued" or "processing"
```

Reply:
Ah yes my bad, I thought you were testing the `get tasks` results where the filter was enqueued and processing